King-Benx/hello_microverse
# update readme

**Description**
This update fixes the past reviewers feedback on updating the readme and also not merging pull requests until they are ready.

**How to Test**
1. Navigate to the main repository.
2. Checkout to the fix readme branch
3. Take note of the readme.
4. You can also set up the project locally to test the application to make sure it is running in your browser by selecting the file and either opening in browser or running live server
```git clone git@github.com:King-Benx/hello_microverse.git``` then ```git checkout fix-readme```

**What has been done so far**
1. Applying Github actions [Github Actions](https://github.com/King-Benx/hello_microverse/pull/2)
2. Applying local linters [Local Linters](https://github.com/King-Benx/hello_microverse/pull/4)
3. Completion of project [html&css project](https://github.com/King-Benx/hello_microverse/pull/5)
5. Removing the uploaded test.md file [Test.md](https://github.com/King-Benx/hello_microverse/pull/6)
